components/cd-utilities/cd-utilities.css

```diff
@@ -0,0 +1,66 @@
+/* Utlilty classes */
+
+/* Alignment */
+.cd-align--center,
+*[class*="--align-center"]  {
+  text-align: center;
+}
+
+.cd-align--right,
+*[class*="--align-right"] {
+  text-align: right;
+}
+
+[dir='rtl'] .cd-align--right,
+[dir='rtl'] *[class*="--align-right"] {
+  text-align: left;
+}
+
+/* Floating */
+.cd-float--left,
+*[class*="--float-left"] {
+  float: left;
+}
+
+[dir='rtl'] .cd-float--left,
+[dir='rtl'] *[class*="--float-left"] {
+  float: right;
+}
+
+.cd-float--right,
+*[class*="--float-right"] {
+  float: right;
+}
+
+[dir='rtl'] .cd-float--right,
+[dir='rtl'] *[class*="--float-right"] {
+  float: left;
+}
+
+/* Rounded image */
+/* Requires square image */
+*[class*="__image--rounded"] {
+  display: block;
+  border-radius: 50%;
+  overflow: hidden;
+  /* For IE11 */
+  width: 100%;
+}
```

Comment:
I found my way here by way of the `cd-byline` and its rounded image.

I wonder if that should be made slightly more generic? just `--rounded` should do, or perhaps if it were its own standalone class: `.cd-util--rounded` — then the class name denotes where it came from, and isn't tied to a particular sub-element (in this case a class containing `__image`).

Most of these rules seem more like global defaults rather than utilities to me. If I've missed out on important reasons why they're here, please bop me on the head 😄  If not, maybe we can discuss on the next call and see what we could do to either make them more generic and utility-like.

Reply:
Yes, this is open for discussion. I've been using the utilities as a place for things I don't want to make into components because they are too small to warrant it, but I don't know where else they should go. I'm happy to rename this component or any selectors to improve clarity.